### DISCUSSION TITLE
Photo disappears after freezing and unfreezing bean

### DISCUSSION THREAD

**Discussion by Ravanger** [graphefruit/Beanconqueror#813] [Q&A] [answered]
When I add a bean including a photo, freeze it, then unfreeze it, the photo is no longer attached to the bean.

I'm using the Android app, version 7.5.0

**Comment by graphefruit** [comment, marked as answer]
Hello @Ravanger ,
are you going over the popup? 

If you want to freeze the whole bag go directly into the edit mode and set the freezing date, then no photo will be lost.

**Comment by Ravanger** [comment]
Hi @graphefruit ,
That seems to work, I'm not sure what I was doing wrong before :)
Thank you!
